### DISCUSSION TITLE
Amazing work

### DISCUSSION THREAD

**Discussion by rd4com** [pthom/imgui_bundle#128] [General]
Hello, congratulation, imgui is indeed important for everybody.
i wonder what do you think about the new mojo language ?
would you contemplate the idea of starting similar bindings in mojo?

great work of love

**Comment by pthom** [comment]
Hello,

Thanks a lot for your feedback! 

Concerning mojo, I have not tried it. However the doc says that it can import any python module. So, maybe imgui_bundle may work with no additional change. Mojo is quite young, though; so there might be some rough edges.
